README.md

```diff
@@ -1,6 +1,7 @@
 # Zowe Explorer
 
 [![codecov](https://codecov.io/gh/zowe/vscode-extension-for-zowe/branch/master/graph/badge.svg)](https://codecov.io/gh/zowe/vscode-extension-for-zowe)
+[![slack](https://img.shields.io/badge/chat-on%20Slack-blue?style=flat-square)](https://slack.openmainframeproject.org/)
```

Comment:
The style of the badges doesn't match:
![image](https://user-images.githubusercontent.com/50887074/81176352-0c624800-8f73-11ea-8505-1cdf3ca7faa5.png)

If `?style=flat-square` is removed from the Slack badge URL, then both badges should have rounded corners.

Reply:
Done.